server/backend/courses/models.py

```
@@ -36,18 +36,34 @@ class CourseContents(models.Model):
     CONTENT_TYPES = [
         ('video', 'Video'),
         ('pdf', 'PDF'),
-        ('text', 'Text'),
         ('article', 'Article'),
     ]
+
     course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='contents')
     content_type = models.CharField(max_length=20, choices=CONTENT_TYPES)
     title = models.CharField(max_length=255)
+    order = models.PositiveIntegerField(default=0)
     url = models.URLField(null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `URLField`

(DJ001)